target_postgres/__init__.py

```diff
@@ -46,7 +46,7 @@ def new_csv_file_entry():
         TemporaryFile(mode='w+b'),
         newline='\n',
         encoding='UTF-8',
-        errors='surrogatepass'
+        errors='replace'
```

Comment:
you had this in https://github.com/Pathlight/target-postgres/pull/13/ and then you replaced in https://github.com/Pathlight/target-postgres/pull/14. any reason to switch back? or maybe we need both (is that possible)?